recipes/lytekit/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "lytekit" %}
+{% set version = "0.15.27" %}
+{% set python_max = "3.11" %}
```

Comment:
Is this from something concretely in the package code that is incompatible with 3.11+? If so, then use it in the `run:` section.

If this was only because dependencies couldn't solve, then I'd recommend removing this:

```suggestion
```

Reply:
The `noarch` strategy only tests `python_min`, so you don't need to block it with `run` requirements. Please remove.

Reply:
@nh13 there are still some review actions to be addressed, like this one.